### PR TITLE
New artefact test modification

### PR DESCRIPTION
**Pull request by n-kai**
This PR will update the following text and make a few minor editorial update

_To utilize new artefacts, they must have approval from the BIO-iTC. The requirements for submitting new artefacts for use can be found at the Biometrics Security home page. If the new artefact type is approved, it will be included as part of the [Toolbox] for use. If the new artefact type is not approved, then only the artefacts already in the [Toolbox] may be used._

### CODE REVIEW

File: Supporting Documents/BS_SD.adoc
Comment (gfiumara) [inline Supporting Documents/BS_SD.adoc:893]:
```suggestion
However, new artefacts species may be found after the <<Toolbox>> is published. The evaluator shall search publicly available information that is published after the publication date of the <<Toolbox>> to look for new artefact species. New artefact species are those artefacts that are out of scope of the <<Toolbox>> and need to be made in a completely different way with significantly different materials that are not covered by the <<Toolbox>>.
```

File: Supporting Documents/BS_SD.adoc
Comment (gfiumara) [inline Supporting Documents/BS_SD.adoc:898]: Can we require this instead?

I think we should also have the evaluator report new types of artefacts including those that cannot successfully attack the TOE because if different enough, they may be able to successfully attack other targets.

```suggestion
The evaluator shall report to the BIO-iTC when new artefact species are found so the artefacts may be added to the <<Toolbox>>. 
```

**Review by woodbe — APPROVED** [review]
My one concern on the new artefacts is that we are still implying that artefacts could be added before approval by the iTC. I think that it needs to be clear that while new artefacts can be added, and even provisionally added quickly (assuming they provide all the information), that if they don't have it approved by the iTC prior to testing, they cannot be considered part of the toolbox. I want this to be explicitly stated so it is not open to interpretation.

**Comment by gfiumara** [comment]
@woodbe Can we address your concern by re-adding the deleted sentence?

 > To utilize new artefacts, they must have approval from the BIO-iTC.

or modifying it somehow, that would also incorporate some of the additional information you make [in this comment](https://github.com/biometricITC/cPP-biometrics/pull/345#issuecomment-868516037)?

 > To utilize new artefacts, they must have approval from the BIO-iTC **through  publication in an incremented toolbox version for the evaluator to evaluate against. [footnote: Publication can occur rapidly, typically ≤2 weeks]**

The conversation is split between the two PRs, but I think the concern is we want evaluators to be able to make new artefact if they want to, but not to claim them as official unless the iTC says so, especially in the event that the device does not pass the evaluation with the new artefact.

**Pull request merged**
